content/posts/2020-02-19-sagas-typing/index.mdx

```diff
@@ -0,0 +1,287 @@
+---
+title: How to type a saga
+author: Fabien Trestour
+date: 2020-02-19
+hero: ./images/header.png
+excerpt: Redux-saga is a very useful tool to have in a front-end stack. Typing them properly is not obvious though. Here's a guide on how to do it!
```

Comment:
In articles, your words weight more when they are factual. `a very useful tool` is not factual, it's an opinion. It's not a problem but it is important to be aware of this. Something like `is a widely used library that help to deal with side effects in a front-end app` is more factual.

Reply:
Maybe speak about typescript 3.7 with a link to the doc to explain that generators can now be typed